docs/experiments-msmarco-passage.md

```diff
@@ -101,6 +102,15 @@ bin/run.sh io.anserini.index.IndexCollection \
   -generator DefaultLuceneDocumentGenerator \
   -threads 9 -storePositions -storeDocvectors -storeRaw 
 ```
+In Windows:
+```bash
+java -cp target\anserini-0.12.0-fatjar.jar io.anserini.index.IndexCollection ^
```

Comment:
yes, sorry for my mistake. Actually, I use anserini v0.37.1 for the windows scripts.

Reply:
I would still like a "launch script" that parallels `bin/run.sh` so that we don't need to change the version of the fatjar every time we do a release.

Reply:
I will write code for it and again make a new pull request for that.